pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -3579,7 +3579,9 @@
       }
     },
     "System.Runtime.Intrinsics.X86": {
-      "InboxOn": {},
+      "InboxOn": {
+        "netcoreapp2.1": "4.0.0.0"
+      },
```

Comment:
The build system requires  "InboxOn". Is this version right?

Reply:
Yes it should be.